librz/core/p/core_java.c

```diff
@@ -1935,15 +1624,67 @@ static bool rz_cmd_java_handle_print_exceptions(RzCore *core, const char *input)
 			free(class_info);
 		}
 	}
-	return true;
+	return RZ_CMD_STATUS_OK;
+}
+
+RZ_IPI RzCmdStatus rz_cmd_java_handler(RzCore *core, int argc, const char **argv) {
+	for (int i = 0; i < END_CMDS; i++) {
+		if (!strncmp(argv[1], JAVA_CMDS[i].name, JAVA_CMDS[i].name_len)) {
+			return JAVA_CMDS[i].handler(core, argc - 2, argc > 2 ? &argv[2] : NULL);
+		}
+	}
+	return RZ_CMD_STATUS_NONEXISTINGCMD;
```

Comment:
this should probably return `WRONG_ARGS`, otherwise executing `java asdasd` returns `Command 'java' does not exist.` which is confusing.

Reply:
oh. i'll fix this tonight.